dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/RouteController.java

```diff
@@ -80,7 +84,7 @@ public ResponseEntity<StreamingResponseBody> run(
       @PathVariable("id") String id,
       @CurrentUser UserDetails currentUser,
       HttpServletRequest request)
-      throws IOException, ForbiddenException, NotFoundException, BadRequestException {
+      throws Exception {
```

Comment:
Please avoid this in controllers. If we can list specific exceptions we have to so OpenAPI docs "sees" them.